postman/Get Broadcast Stats.postman_collection.json

```diff
@@ -0,0 +1,569 @@
+							"    pm.response.to.have.status(200);",
+							"});",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{listId}}",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Total Sent Broadcasts",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "e238f70c-3010-41a2-ba60-6fd4e312850d",
+						"exec": [
+							"var jsonData = pm.response.json();",
+							"pm.globals.set(\"broadcast_self_link\", '');",
+							"",
+							"pm.test(\"Successfully retreived sent broadcasts from list\", function() {",
+							"    if (pm.expect(jsonData.entries.length).to.be.above(0)) {",
+							"        var broadcast_entry =  Math.floor((Math.random() * jsonData.entries.length ));",
+							"        pm.globals.set(\"broadcast_self_link\", jsonData.entries[broadcast_entry].self_link );",
+							"    }",
+							"});",
+							"",
+							"pm.test(\"Successfully submitted request\", function(){",
+							"    pm.response.to.have.status(200);",
+							"});",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{sent_broadcasts_link}}",
+					"host": [
+						"{{sent_broadcasts_link}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get a Broadcast",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "c8c9f482-117c-4cc9-87c7-04dbb1e65bfb",
+						"exec": [
+							"pm.test(\"Successfully submitted request\", function(){",
+							"    pm.response.to.have.status(200);",
+							"});",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{broadcast_self_link}}",
+					"host": [
+						"{{broadcast_self_link}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Stats Entries",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "c51162ab-dfa9-42e5-82da-3acf6ca35bd1",
+						"exec": [
+							"pm.test(\"Successfully submitted request\", function(){",
+							"    pm.response.to.have.status(200);",
+							"});",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{listId}}/campaigns/b{{broadcastId}}/stats",
```

Comment:
i dont see where **broadcastId** is being set

Reply:
Updated